# Physical parameters
c : 1.484 # speed of sound in medium
receiver_locations: # position of non-reference receivers (reference (origin) not included)
  - [ 25.4,    0,  0]
  - [-25.4,    0,  0]
  - [    0, 25.4,  0]

# Input parameters
input_mode : paul_board
input_timeout : 1.0
pinger_freq : 75000
sampling_freq : 300000
locating_frame : "map"
receiver_array_frame : "map"
#receiver_array_frame : "hydrophones"

# Paul Board parameters
port : "/dev/serial/by-id/usb-FTDI_FT232R_USB_UART_A901AN8N-if00-port0"
baud : 115200
permute : "1 2 3 4"
dist_h : 2.286000e-02
dist_h4 : 2.286000e-02
v_sound : 1482

# Algorithm parameters
method : "LS"
target_frequency : 35000
upsampling_factor : 20
min_variance : 200
observation_buffer_size : 200

